doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -56,6 +59,7 @@ server {
     {
         add_header 'Access-Control-Allow-Origin' '*';
         alias /usr/share/jitsi-meet/$1/$2;
+        expires 1y;
```

Comment:
Won't this create trouble with updates?

Reply:
I see that most static files are versioned already with ``?v=``. But hypothetically it could cause some issues, for example if the mp3s are updated.

Reply:
or watermark.png, which I guess will be updated more often than the mp3s and wavs if people switch to their own logo.

Reply:
Any chance you could limit this to urls having v= in their args?

Reply:
Good idea, changed it now